src/components/MediaList/MediaList.tsx

```diff
@@ -160,21 +165,33 @@ const isLink = (url: string) => url.startsWith("http") || url.startsWith("https"
 interface HandleLinkProps {
   url: string;
   children: ReactNode;
+  type: ItemTypeWithDescription["metaType"];
 }
 
-export const HandleLink = ({ url, children }: HandleLinkProps) => {
+const licenceTag = (type: ItemTypeWithDescription["metaType"], url: boolean): string | undefined =>
+  ({
+    title: "dct:title",
+    author: "cc:attributionName",
+    copyrightHolder: "cc:copyrightHolder",
+    contributor: "cc:contributor",
+    other: url ? "cc:attributionURL" : undefined,
+    //@ts-ignore
+  })[type];
+
+export const HandleLink = ({ url, children, type }: HandleLinkProps) => {
+  const tag = licenceTag(type, isLink(url));
+
   if (isLink(url)) {
     return (
-      <SafeLink to={url} target="_blank" rel="noopener noreferrer">
+      <SafeLink to={url} target="_blank" rel={`noopener noreferrer ${tag}`}>
```

Comment:
Det er vel ikke sånn rel funker på anchors?

Reply:
Litt lenge siden jeg la til denne så jeg husker ikke helt, men det var et eksempel der dette ble brukt sånn. Skal prøve å finne den

Reply:
Her tror jeg at jeg prøvde å lage noe fornuftig basert på denne regelen: 
- rel="cc:attributionURL" generates a new triple with predicate cc:attributionURL, and the URL in the href as the object. The attribution URL is important when you want to indicate what URL re-users of your CC-licensed work should link to when they attribute your work. This URL should appear in the license deed when someone clicks on your link to the CC license.
https://opensource.creativecommons.org/ccrel-guide/ under Attribution metadata

Reply:
Synes det virker rimelig!